.dockerignore

```diff
@@ -0,0 +1,7 @@
+.git*
+db/*.sqlite3
```

Comment:
If we're switching to `postgres` completely this line and the following one should no longer be required (I think).